platform-service/src/unstract/platform_service/controller/platform.py

```diff
@@ -110,6 +117,7 @@ def validate_bearer_token(token: Optional[str]) -> bool:
         cursor = be_db.execute_sql(query)
         result_row = cursor.fetchone()
         cursor.close()
+        be_db.manual_close()
```

Comment:
@harini-venkataraman to my knowledge we open this connection in line 38, if we close this here can you confirm how we reopen the connection?

Reply:
Also not sure if we even need the connect and close when using pooled connection. @harini-venkataraman could you confirm?

Reply:
Checking at the docs I don't see usage of connect in that. Can you check and confirm?

Reply:
As per, peewees' documentation, it is best practice to close the connection ( i.e the connection it returned back to the pool for reuse ), and it is up to the application developer to gracefully handle this. Have changed `manual_close()` which does not return to pool to `close()`
Ref : https://docs.peewee-orm.com/en/latest/peewee/playhouse.html#PooledDatabase
https://github.com/coleifer/peewee/issues/1143#issuecomment-266308713
Quote from Docs :
> By default, all your application needs to do is ensure that connections are closed when you are finished with them, and they will be returned to the pool. For web applications, this typically means that at the beginning of a request, you will open a connection, and when you return a response, you will close the connection.